FogCreek/Glitch-Community
# Tiny capybara: componentize DataLoader

## Links
* Remix link: https://tiny-capybara.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328384/

## Changes:
* rewrite DataLoader with hooks
* move to components directory
* rename loaders/loader -> loader
* (no storybook because no user interface)

## How To Test:
* visit https://tiny-capybara.glitch.me/~community
* the project page & readme should load
* visit https://tiny-capybara.glitch.me/~sdfhafgjkaslhdgf
* it should display "We didn't find sdfhafgjkaslhdgf"
